.storybook/recipes/Filters/Filters.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import { StoryObj, Meta } from '@storybook/react';
+import React from 'react';
+import { Filters } from './Filters';
+import Checkbox from '../../../src/components/Checkbox';
+import FiltersCheckboxField from '../../../src/components/FiltersCheckboxField';
+
+export default {
+  title: 'Recipes/Filters',
+  component: Filters,
+  args: {
+    as: 'FiltersDrawer',
+    hasSelectedFilters: false,
+    placement: 'bottom-start',
+    triggerText: 'Filters',
+    children: (
+      <FiltersCheckboxField legend="Filters Segment 1">
+        <Checkbox
+          label="Filters label 1"
+          onChange={
+            () => {} /* eslint-disable-line @typescript-eslint/no-empty-function */
+          }
+        />
+        <Checkbox
+          label="Filters label 2"
+          onChange={
+            () => {} /* eslint-disable-line @typescript-eslint/no-empty-function */
+          }
+        />
+        <Checkbox
+          label="Filters label 3"
+          onChange={
+            () => {} /* eslint-disable-line @typescript-eslint/no-empty-function */
+          }
+        />
+      </FiltersCheckboxField>
+    ),
+  },
+  decorators: [
+    (Story) => (
+      <div style={{ margin: '0.25rem', height: '100vh' }}>
+        <Story />
+      </div>
+    ),
+  ],
+} as Meta<Args>;
+
+type Args = React.ComponentProps<typeof Filters>;
+
+export const Default: StoryObj<Args> = {};
```

Comment:
Good question, I left it as one story so that whoever views the story can choose the storybook control for FiltersPopover. Didn't think there was much value in Chromatic testing (flakiness and already being tested in FiltersPopover stories). Also this way, the dev can know to implement in a similar manner.

But now looking back at it, not sure how many devs use the control so I'll make the second story.

Reply:
>not sure how many devs use the control

Yeah that's what I'm thinking too 👍